x/crosschain/types/cmd_cctxs.go

```diff
@@ -304,6 +308,8 @@ func newCmdCCTX(
 				GasPrice:       medianGasPrice,
 				GasPriorityFee: priorityFee,
 				TssPubkey:      tssPubKey,
+				// use same confirmation mode as inbound, will be overwritten by actual confirmation mode
```

Comment:
What does it mean? Let's use SAFE for all CMD tx

Reply:
https://github.com/zeta-chain/node/pull/3506/commits/73fa3df5c5d0065c944d0ddcf62bf10219b86d7b